src/ui/public/agg_response/tabify/_buckets.js

```diff
@@ -38,7 +38,10 @@ function TabifyBuckets(aggResp, aggParams) {
     this.length = this.buckets.length;
   }
 
-  if (this.length && aggParams) this._orderBucketsAccordingToParams(aggParams);
+  if (this.length && aggParams) {
+    this._orderBucketsAccordingToParams(aggParams);
+    if (aggParams.drop_partials) this._dropPartials(aggParams, timeRange);
```

Comment:
nit: it's better to align with our styleguide always-use-braces-for-conditionals-and-loops